linux/installer/common/psw/Makefile

```diff
@@ -39,13 +38,14 @@ SOURCE_INSTALL_PATH=$(INSTALL_PATH)
 SCRIPTS_INSTALL_PATH=$(INSTALL_PATH)/$(SCRIPTS_FOLDER)
 PSW_LIB_PATH=$(SOURCE_INSTALL_PATH)/$(LIB_DIR)
 USR_LIB_PATH=$(addprefix $(DESTDIR)/usr/, $(or $(notdir $(shell gcc -print-multi-os-directory 2> /dev/null)),$(if $(wildcard /usr/lib64/.*),lib64),lib))
-
 SPLIT_VERSION=$(word $2,$(subst ., ,$1))
 
 ECL_VER=1.0.0
 LCH_VER=1.0.0
 EPID_VER=1.0.0
 QEX_VER=1.0.0
+URTS_FULL_VER:= 2.0.100.3
```

Comment:
`URTS_VER:= 2.0.0` is enough for bin installer. This version is not updated until major version changes.